rascal2/widgets/sliders_view.py

```diff
@@ -0,0 +1,550 @@
+        #          but benefit of that  is minuscules.
+        # slider_name
+        #       -- name of the property, slider describes and key, which defines slider position in the dictionary
+        #          of sliders
+        """
+        if self.isVisible():  # Do not bother otherwise, as slider appearance will be modified when made visible and
+            # slider itself may have even been deleted
+            self._sliders[slider_name].update_slider_display_from_property(in_constructor=False)
+
+    def _create_slider_view_layout(self) -> None:
+        """Create sliders layout with all necessary controls and connections
+        but without sliders themselves.
+        """
+
+        main_layout = QtWidgets.QVBoxLayout()
+
+        accept_button = QtWidgets.QPushButton("Accept", self, objectName="AcceptButton")
+        accept_button.clicked.connect(self._apply_changes_from_sliders)
+
+        cancel_button = QtWidgets.QPushButton("Cancel", self, objectName="CancelButton")
+        cancel_button.clicked.connect(self._cancel_changes_from_sliders)
+
+        button_layout = QtWidgets.QHBoxLayout()
+        button_layout.setAlignment(QtCore.Qt.AlignmentFlag.AlignRight)
+        button_layout.addWidget(accept_button)
+        button_layout.addWidget(cancel_button)
+
+        main_layout.addLayout(button_layout)
+
+        self.setLayout(main_layout)
+
+    def _add_sliders_widgets(self) -> None:
+        """Given sliders view layout and list of properties which can be controlled by sliders
+        add appropriate sliders to sliders view Widget
+        """
+        scroll = self.findChild(QtWidgets.QScrollArea, "Scroll")
+        if scroll is None:
+            main_layout = self.layout()
+            scroll = QtWidgets.QScrollArea()
+            scroll.setWidgetResizable(True)  # important: resize content to fit area
+            scroll.setObjectName("Scroll")
+            main_layout.addWidget(scroll)
+            content = QtWidgets.QWidget()
+            content.setObjectName("Scroll_content")
+            scroll.setWidget(content)
+            # --- Add content layout
+            content_layout = QtWidgets.QVBoxLayout(content)
+        else:
+            content = scroll.findChild(QtWidgets.QWidget, "Scroll_content")
+            content_layout = content.layout()
+
+        # We are adding new sliders, so delete all previous ones. Update is done in another branch.
+        for slider in self._sliders.values():
+            slider.deleteLater()
+        self._sliders = {}
+
+        if len(self._prop_to_change) == 0:
+            no_label = EmptySlider()
+            content_layout.addWidget(no_label)
+            self._sliders[no_label.slider_name] = no_label
+        else:
+            content_layout.setSpacing(0)
+            for name, prop in self._prop_to_change.items():
+                slider = LabeledSlider(prop)
+
+                self._sliders[name] = slider
+                content_layout.addWidget(slider)
```

Comment:
You need to add a stretch at the end of layout somewhere, otherwise ui looks like this with single slider

<img width="834" height="367" alt="Image" src="https://github.com/user-attachments/assets/ccf69e7a-4374-47e4-9954-a1c7185efd04" />

Reply:
I will accept MaximumHeight as a temporary fix, I will refactor in future. Maximum height can lead to clipping of content  so I try to avoid when possible